Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogScreen.h

```diff
@@ -44,6 +44,10 @@ namespace O3DE::ProjectManager
         bool IsTab() override;
         void NotifyCurrentScreen() override;
 
+        void AddToGemModel(GemInfo gemInfo);
+
+        void ShowStandardToastNotification(QString notification);
```

Comment:
both of these should use const ref to avoid unnecessary copying of the parameter
```suggestion
        void AddToGemModel(const GemInfo& gemInfo);

        void ShowStandardToastNotification(const QString& notification);
```